flow/activities/flowable.go

```diff
@@ -509,6 +515,8 @@ func (a *FlowableActivity) GetQRepPartitions(ctx context.Context,
 		}
 	}
 
+	a.Alerter.LogFlowInfo(ctx, config.FlowJobName, fmt.Sprintf("obtained partitions for table %s", config.WatermarkTable))
```

Comment:
Maybe just log instead of sending an alert? (too noisy)

Reply:
These are not alerts. This is for our logs table in UI
I think `LogFlowEvent` is for alerts ?

Reply:
My bad, but still do we want to do it for all tables?

Reply:
yeah because we have scarcity of user facing info during initial load - common feedback from customers cc @iskakaushik